### PR TITLE
Ensure CarePlan is created with valid fields

### PR DESCRIPTION
**Pull request by dwelman**

### CODE REVIEW

**Review by reinkrul — CHANGES_REQUESTED** [review]
`CarePlan.status` must be `active`

**Comment by dwelman** [comment]
> `CarePlan.status` must be `active`

It is: `orchestrator/careplanservice/handle_createtask.go:88` 
`carePlan.Status = fhir.RequestStatusActive`

File: orchestrator/careplanservice/handle_createtask.go [hunk 1]
```diff
@@ -69,13 +69,24 @@ func (s *Service) handleCreateTask(ctx context.Context, request FHIRHandlerReque
...
 				Reference: to.Ptr(taskURL),
 				Type:      to.Ptr(coolfhir.ResourceType(task)),
 			},
 		})
+
+		// Validate Task.For
+		if task.For == nil || task.For.Identifier == nil {
+			return nil, coolfhir.NewErrorWithCode("Task.For must be set with a reference to a patient", http.StatusBadRequest)
+		}
+		if *task.For.Identifier.System != "http://fhir.nl/fhir/NamingSystem/bsn" {
```
Comment (reinkrul) [inline orchestrator/careplanservice/handle_createtask.go:84]:
```suggestion
		if *task.For.Identifier.System != coolfhir.BSNNamingSystem {
```

**Review by reinkrul — APPROVED** [review]
Small improvement

**Review by reinkrul — APPROVED** [review]

**Pull request merged**